openxr/src/space.rs

```diff
@@ -130,6 +130,109 @@ impl Space {
         ))
     }
 
+    /// Determine the locations of the joints of a hand tracker relative to this space at a
+    /// specified time, if currently known by the runtime.
+    ///
+    /// XR_EXT_hand_tracking must be enabled.
+    #[inline]
+    pub fn locate_hand_joints(
+        &self,
+        tracker: &HandTracker,
+        time: Time,
+    ) -> Result<Vec<HandJointLocation>> {
```

Comment:
We should also be returning `is_valid` somehow

Reply:
Would it be sensible to return an `Option` with `None` when invalid, rather than returning a bunch of data with undefined values?

Reply:
Yeah, that can work.